### ISSUE TITLE
Enigmatic Amulet Won't create Death item portal with some case and remove every item

### ISSUE THREAD

**Issue by EVSkyFall** [Extegral/Enigmatic-Legacy#488]
#### Versions (Be specific, do not write "latest"):1.19.2
 * Enigmatic Legacy:2.26.5
 * Curios API:5.1.4.4
 * Patchouli:2-77
 * Forge:43.3.5
 * Iter RPG:0.5.0
 * Create:0.5.1.f
 * Modpack:Steampunk[LPS] v21.5

#### Description of a Bug:

If you killed by "Create" mod's "Deployer" using "Bloodranch" from "Iter RPG"  while wearing enigmatic amulet, it won't create death inventory portal and won't drop any item you held.
basically remove every single item you had

#### How to Reproduce:
1. Equip Enigmatic Amulet
2. Set deployer and give it bloodranch
3. Get killed by bloodranch projectile
4. Behold!

#### Logs:

Crash Log: Please use a paste site like [Github Gists](https://gist.github.com).
